docs/reference/release-notes/highlights-7.4.0.asciidoc

```diff
@@ -0,0 +1,106 @@
+[[release-highlights-7.4.0]]
+== 7.4.0 release highlights
+++++
+<titleabbrev>7.4.0</titleabbrev>
+++++
+
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+[float]
+==== Results pinning
+
+With the release of Elasticsearch 7.4.0, we provide the ability to define the first records in a result set (and the order in which they will show up) directly within Elasticsearch. This capability is provided by the new https://www.elastic.co/guide/en/elasticsearch/reference/7.4/query-dsl-pinned-query.html[pinned query].
```

Comment:
I don't feel that we need to mention the 7.4.0 release in this item as we're already on the release highlights page.

```suggestion
You can use the new <<query-dsl-pinned-query, pinned query>>
to define the first records
(and the order in which they are returned)
in a result set directly within {es}.
```